src/lib/utils.js

```diff
@@ -56,4 +56,29 @@ export function combine(...objects) {
     });
 
     return result;
+}
+
+/**
+ * Check whether given object is a number.
+ * 
+ * @param {object} numberToTest 
+ * @returns {boolean}
+ */
+export function isNumber(numberToTest){
+    return !isNaN(parseFloat(numberToTest)) && isFinite(numberToTest);
+}
+
+/**
+ * Converts array to object
+ * @param {Array.<object>} array
+ * @returns {object}
+ */
+export function arrayToObject(array){
+    let result = {};
```

Comment:
Should be `const` instead of `let`. However, if you used `reduce` you would not need to declare any variables.